keel-sql/keel-sql.gradle.kts

```diff
@@ -1,16 +1,22 @@
 import ch.ayedo.jooqmodelator.gradle.JooqModelatorTask
 import com.diffplug.gradle.spotless.SpotlessExtension
 
+val buildingInDocker = project.properties["buildingInDocker"]?.toString().let { it == "true" }
+
 plugins {
   `java-library`
   id("kotlin-spring")
   id("ch.ayedo.jooqmodelator") version "3.9.0"
+  id("nu.studer.jooq") version "5.0.1"
```

Comment:
We didn't strictly need this plugin, but I used it to generate the list of runtime dependencies for the `jooqGenerate` custom task below because I suck at gradle. I tried creating a vanilla configuration instead but got stuck. Would be happy to remove the plugin if anyone can show me the way.